sumologic/resource_sumologic_monitors_library_monitor.go

```diff
@@ -553,17 +553,45 @@ var sloBurnRateTriggerConditionSchema = map[string]*schema.Schema{
 		"time_range": &timeRangeSchema,
```

Comment:
this should also be optional, right ?

Reply:
yes, updated in this [commit](https://github.com/SumoLogic/terraform-provider-sumologic/pull/499/commits/9293f3b1f3677ab364fe058c92506f1aaa133c3d)